decidim-meetings/db/migrate/20240130135858_add_withdrawn_fields_on_meetings.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class AddWithdrawnFieldsOnMeetings < ActiveRecord::Migration[6.1]
+  class CustomMeeting < Decidim::Meetings::ApplicationRecord
+    self.table_name = "decidim_meetings_meetings"
+  end
+
+  def up
+    add_column :decidim_meetings_meetings, :withdrawn_at, :datetime
+
+    CustomMeeting.where(state: "withdrawn").find_each do |meeting|
+      meeting.withdrawn_at = meeting.updated_at
+      meeting.save!
+    end
+  end
+
+  def down
+    CustomMeeting.where.not(withdrawn_at: null).find_each do |meeting|
+      meeting.state = :withdrawn
+      meeting.save!
+    end
```

Comment:
Using `find_each` is more expensive that a sql query. Considering that you do not need to set a row specific value, then `update_all` would be the way to go. 
```suggestion
    CustomMeeting.where.not(withdrawn_at: null).update_all(state: :withdrawn)
```

Reply:
Got it!

Reply:
Looks like Rubocop isn't a fan of that change because it skips the validation. Shall we revert it back? Or disable/ignore this line? https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code